_inc/client/security/antispam.jsx

```diff
@@ -3,36 +3,54 @@
  */
 import React from 'react';
 import { translate as __ } from 'i18n-calypso';
+import TextInput from 'components/text-input';
+import FoldableCard from 'components/foldable-card';
 
 /**
  * Internal dependencies
  */
-import { FormFieldset } from 'components/forms';
+import { FormFieldset, FormLabel } from 'components/forms';
 import { ModuleSettingsForm as moduleSettingsForm } from 'components/module-settings/module-settings-form';
-import { ModuleSettingCheckbox } from 'components/module-settings/form-components';
 import SettingsCard from 'components/settings-card';
 import SettingsGroup from 'components/settings-group';
 
 export const Antispam = moduleSettingsForm(
 	React.createClass( {
 
 		toggleModule( name, value ) {
-			this.props.updateFormStateOptionValue( name, !value );
+			this.props.updateFormStateOptionValue( name, ! value );
 		},
 
 		render() {
 			return (
 				<SettingsCard
 					{ ...this.props }
 					header={ __( 'Spam filtering', { context: 'Settings header' } ) }>
-					<SettingsGroup support="https://akismet.com/jetpack/">
-						<FormFieldset>
-							<ModuleSettingCheckbox
-								name={ 'akismet_show_user_comments_approved' }
-								{ ...this.props }
-								label={ __( 'Show the number of approved comments beside each comment author' ) } />
-						</FormFieldset>
-					</SettingsGroup>
+					<FoldableCard
+						header={ __( 'Your site is protected from spam' ) }
+					>
+						<SettingsGroup support="https://akismet.com/jetpack/">
+							<FormFieldset>
+								<FormLabel>
+									<span className="jp-form-label-wide">{ __( 'Your API key' ) }</span>
+									<TextInput
+										name={ 'wordpress_api_key' }
+										value={ this.props.getOptionValue( 'wordpress_api_key' ) }
+										disabled={ this.props.isSavingAnyOption( 'wordpress_api_key' ) }
+										onChange={ this.props.onOptionChange } />
+								</FormLabel>
+								<p className="jp-form-setting-explanation" >
```

Comment:
Would it be a good idea to maybe hide this if they already have a valid api key?